# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1424750730 18000
#      Mon Feb 23 23:05:30 2015 -0500
# Node ID 4006a7377a190ef6b975574f5c2038b497a3d294
# Parent  d3dcc62512d786d89042c2014bbf227d1cd3dcc3
don't save and restore FFLAGS in configure (bug #44359)

* configure.ac: Don't attempt to save and restore FFLAGS.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -311,26 +311,24 @@ AC_PROG_CXX
 AC_PROG_CXXCPP
 
 ### Determine which C compiler to use (we expect to find gcc).
 
 AC_PROG_CC
 AC_PROG_CPP
 AC_PROG_GCC_TRADITIONAL
 
-## Save and restore CFLAGS, CXXFLAGS, and FFLAGS globally.  These variables
-## are for users, so we shouldn't be touching them.  Instead, we should set
-## the corresponding AM_ flags (currently by way of the XTRA_ variables).
-## However, for the duration of the configure script, we may set CFLAGS,
-## CXXFLAGS, or FFLAGS so that subsequent tests succeed.  Temporary
-## settings like this are currently done for pthreads and openmp, for
-## example.
+## Save and restore CFLAGS and CXXFLAGS globally.  These variables
+## are for users, so we shouldn't be touching them.  Instead, we should
+## set the corresponding AM_ flags (currently by way of the XTRA_ variables).
+## However, for the duration of the configure script, we may set CFLAGS
+## and CXXFLAGS so that subsequent tests succeed.  Temporary settings
+## like this are currently done for pthreads and openmp, for example.
 original_octave_configure_CFLAGS="$CFLAGS"
 original_octave_configure_CXXFLAGS="$CXXFLAGS"
-original_octave_configure_FFLAGS="$FFLAGS"
 
 ## Check for MSVC
 have_msvc=no
 case $host_os in
   msdosmsvc)
     have_msvc=yes
   ;;
   mingw*)
@@ -2964,17 +2962,16 @@ AM_CONDITIONAL([AMCOND_HAVE_QSCINTILLA],
                [test "$octave_cv_lib_qscintilla" = yes])
 AM_CONDITIONAL([WIN32_TERMINAL], [test $win32_terminal = yes])
 AC_SUBST(QT_CPPFLAGS)
 AC_SUBST(QT_LDFLAGS)
 AC_SUBST(QT_LIBS)
 
 CFLAGS="$original_octave_configure_CFLAGS"
 CXXFLAGS="$original_octave_configure_CXXFLAGS"
-FFLAGS="$original_octave_configure_FFLAGS"
 
 AH_BOTTOM([#include "oct-conf-post.h"])
 
 ### Make all AC_DEFINES available to testif feature of test.m function.
 ### This must reside at the bottom of configure.ac after all AC_DEFINES
 ### have been made.
 
 AC_OUTPUT_MAKE_DEFS
